projects/canopy/src/lib/brand-icon/brand-icon.registry.ts

```diff
@@ -8,17 +10,22 @@ import { BrandIcon, BrandIconName } from './brand-icons.interface';
 export class LgBrandIconRegistry {
   private registry = new Map<BrandIconName, string>();
 
-  registerBrandIcon(icons: Array<BrandIcon>): void {
-    icons.forEach(icon => {
-      this.registry.set(icon.name, icon.data);
-    });
-  }
+  /**
+   * @deprecated The method should not be used as the brand icons are now automatically registered.
+   */
+  registerBrandIcon(icons: Array<BrandIcon>): void {}
 
-  getBrandIcon(name: BrandIconName): string | undefined {
+  async getBrandIcon(name: BrandIconName): Promise<string | undefined> {
```

Comment:
Isn't this a breaking change?

Reply:
As discussed, I've updated the PR to be a breaking change.